core/unix/os_public.h

```diff
@@ -42,6 +42,10 @@
 #ifndef _OS_PUBLIC_H_
 #define _OS_PUBLIC_H_ 1
 
+#ifdef LINUX
+# include "include/sigcontext.h"
```

Comment:
Hmm, could we remove the signal.h from globals_shared.h as well, if we're going to put this here?